### PR TITLE
feat: make instagoric independent of namespace name

### PR DESCRIPTION
**Pull request by michaelfig**
Don't explicitly pass `-n instagoric`... let the user set proper config like `kubectl config set-context --current --namespace=<ns>`.

This allows for multiple Instagoric namespaces per cluster.

### CODE REVIEW

File: bases/rpcnodes/service.yaml [hunk 1]
```diff
@@ -11,20 +11,30 @@ spec:
...
     type: HTTP
 ---
+apiVersion: cloud.google.com/v1
+kind: BackendConfig
+metadata:
+  name: rpc-backendconfig
+spec:
+  healthCheck:
+    port: 26657
+    requestPath: /healthz
```
Comment (toliaqat) [inline bases/rpcnodes/service.yaml:28]: I tried this `localhost:26656/healthz` on a3p but got a 404. What service is exposing /healthz ?

**Review by toliaqat — APPROVED** [review]
Left a comment about /healthz endpoint.

File: bases/rpcnodes/service.yaml (hunk 1 above)
Comment (michaelfig) [reply bases/rpcnodes/service.yaml:28]: Should be http://localhost:26657/healthz

Comment (michaelfig) [reply bases/rpcnodes/service.yaml:28]: Actually, should be http://localhost:26657/abci_info

**Pull request merged**